vocone/vocone.go

```diff
@@ -296,7 +296,7 @@ func startAPI(host string, port int, path string) (*rpcapi.RPCAPI, error) {
 	if err := signer.Generate(); err != nil {
 		return nil, err
 	}
-	httpRouter := httprouter.HTTProuter{}
+	httpRouter := httprouter.HTTProuter{PrometheusID: "vocone-chi"}
```

Comment:
Why "vocone-chi"? I'd not set a custom ID if not necessary (when Grafana is configured, this ID is relevant).